steps/curl-ping/2.0.1/step.yml

```diff
@@ -0,0 +1,39 @@
+title: Ping an URL with cURL
+summary: Ping an URL using cURL. You can also add optional parameters.
+description: |-
+  Performs a [cURL](http://curl.haxx.se/) request on the given URL
+  and reports the result (success/failed) of the response.
+website: https://github.com/bitrise-steplib/steps-curl-ping
+source_code_url: https://github.com/bitrise-steplib/steps-curl-ping
+support_url: https://github.com/bitrise-steplib/steps-curl-ping/issues
+published_at: 2020-09-29T11:42:27.564863664Z
+source:
+  git: https://github.com/bitrise-steplib/steps-curl-ping.git
+  commit: fec960e84977fb5c5fb54bcc0292eb27283f8e03
+host_os_tags:
+- osx-10.9
+type_tags:
```

Comment:
Invalid type tags, please set `project_type_tags` for `web` and `utility` for `type_tags`